change/@azure-msal-node-2020-10-13-17-19-12-node-e2e-tests.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "prerelease",
+  "comment": "Export Node Cache Serializer for use in end-to-end testing framework (#2414)",
+  "packageName": "@azure/msal-node",
+  "email": "hemoral@microsoft.com",
+  "dependentChangeType": "patch",
```

Comment:
```suggestion
  "dependentChangeType": "none",
```

Reply:
I set it to patch because there is a change to the msal-node index file (I exported the Serializer which wasn't being exported before). Does that still qualify as none?

Reply:
`dependentChangeType` should always be `none` for now. The `type` field can stay `prerelease`